lib/puppet/http/service/compiler.rb

```diff
@@ -69,6 +69,9 @@ def get_node(name, environment:, configured_environment: nil, transaction_uuid:
   # @param [String] environment The name of the environment we are operating in
   # @param [String] configured_environment Optional, the name of the configured
   #   environment. If unset, `environment` is used.
+  # @param [Boolean] if true, request that the server check if our `environment`
+  #   matches the server specified environment. If they are mismatched, it will
+  #   return an empty catalog with the server specified environment.
```

Comment:
Should probably move this `@param` after `checksum_type` and also name it

Reply:
I moved the `check_environment` after the `configured_environment`, and updated the docs to match the same order.